host_v2/src/components/GameInProgressContent.tsx

```diff
@@ -72,8 +60,9 @@ export default function PlaceholderContentArea({
       </Grid>
       <Grid item xs={12} sm={4} sx={{ width: '100%', height: '100%' }}>
         <ScrollBoxStyled>
-          <Card />
-          <Card />
```

Comment:
We can add the featured mistakes card here (and in the other locations in this file where I've commented this)